spot_driver/src/robot_state/state_middleware_handle.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2024 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#include <spot_driver/robot_state/state_middleware_handle.hpp>
```

Comment:
Which headers do you mean specifically?

If you're referring to the includes for the ROS messages, the `state_middleware_handle.hpp` header does use those, since that's where we create the publishers, like `std::shared_ptr<rclcpp::Publisher<spot_msgs::msg::BatteryStateArray>> battery_states_publisher_;` and others. Let me know if I'm misunderstanding you.

Reply:
I think the header I was referring to had to do with the `rclcpp::QoS`

Reply:
I added the include for the rclcpp::QoS type.